### PR TITLE
Rate Helpers - SwapRateHelper/DepositRateHelper to take in SimpleQuote

### PR DESCRIPTION
**Pull request by rimrockcapital**
All rate helpers have been configured to take in SimpleQuotes instead of doubles. This will help with consistency (FRARateHelper, FuturesRateHelper both take in SimpleQuote) and is necessary when passing instruments to bucketAnalysis (SimpleQuotes from term structures need to be passed in), which will be added later on.

### CODE REVIEW

File: quantlib/termstructures/yields/rate_helpers.pyx [hunk 1]
```diff
@@ -116,13 +116,14 @@ cdef class SwapRateHelper(RelativeDateRateHelper):
...
+    def from_tenor(cls, Quote rate, Period tenor,
         Calendar calendar, Frequency fixedFrequency,
         BusinessDayConvention fixedConvention, DayCounter fixedDayCount,
         IborIndex iborIndex, Quote spread=None, Period fwdStart=None):
-
+        
+        cdef Handle[_qt.Quote] rate_handle = Handle[_qt.Quote](deref(rate._thisptr))
         cdef Handle[_qt.Quote] spread_handle
-        cdef Handle[_qt.Quote] _rate
+#        cdef Handle[_qt.Quote] _rate    #from merge w/ master, might not need this any longer?
```
Comment (dpinte) [inline quantlib/termstructures/yields/rate_helpers.pyx]: can you delete commented code?

File: quantlib/termstructures/yields/rate_helpers.pyx [hunk 2]
```diff
@@ -69,25 +69,25 @@ cdef class RelativeDateRateHelper:
 
 
 cdef class DepositRateHelper(RateHelper):
-    """Rate helper for bootstrapping over deposit rates. """
+    """Rate helper for bootstrapping over deposit rates. [uses SimpleQuotes] update 05/14/2015"""
```
Comment (dpinte) [inline quantlib/termstructures/yields/rate_helpers.pyx]: I would suggest moving the comment about SimpleQuotes to the changelog (https://github.com/enthought/pyql/blob/master/CHANGES.txt)

**Comment by dpinte** [comment]
@rimrockcapital Thanks for the PR. It looks good with some minor nits. 

I'm looking into the Travis-CI issue. It sounds like a configuration problem.

**Comment by dpinte** [comment]
@rimrockcapital Can you merge master into this branch to get the travis-ci tests to run? I've fixed the configuration issue.

**Comment by tnicolas** [comment]
Didrik,

  checking in.  Have we done everything correctly on our PR (#89)?



Tous ce passe bien en Belgique?


________________________________

**Comment by dpinte** [comment]
@tnicolas @rimrockcapital great, thanks for the updates!

**Pull request merged**